warp10/src/main/java/io/warp10/script/functions/GET.java

```diff
@@ -85,4 +100,37 @@ public static int computeAndCheckIndex(NamedWarpScriptFunction func, int index,
 
     return index;
   }
+
+  /**
+   * Recursively get elements of a nested list given a list of indices. At the last recursive call, a single element is returned.
+   *
+   * For example, we will have:
+   * recNestedGet(func, nestedList, [ 0, 1, 2 ])
+   * -> recNestedGet(func, nestedList[0], [ 1, 2 ])
+   * -> recNestedGet(func, (nestedList.get(0)).get(1), [ 2 ])
+   * -> return ((nestedList.get(0)).get(1)).get(2)
+   *
+   * @param func        The function that calls this method
+   * @param nestedList  The list to which to get the elements
+   * @param indexList   The list of indices.
+   * @return
+   * @throws WarpScriptException
+   */
+  static Object recNestedGet(NamedWarpScriptFunction func, List nestedList, List<Number> indexList) throws WarpScriptException {
```

Comment:
From PR#463, hbs:"This method should not need the func attribute, the WarpScriptException should be caught by the caller which will wrap it and add its name"

Reply:
I think it needs the func attribute in case it is used by another function to have a better error message.

Reply:
Please remove the func attribute.